records_mover/db/bigquery/loader.py

```diff
@@ -106,13 +107,18 @@ def load(self,
 
         if directory.scheme != 'gs':
             with self.temporary_gcs_directory_loc() as temp_gcs_loc:
-                gcs_directory = directory.copy_to(temp_gcs_loc)
-                return self.load(schema=schema,
-                                 table=table,
-                                 load_plan=load_plan,
-                                 directory=gcs_directory)
-
-        logger.info("Loading from records directory into BigQuery")
+                copy_optimizer = CopyOptimizer()
```

Comment:
See records_mover/url/optimizer/__init__.py for info on these methods and why they're used.